ruby/vampires.rb

```diff
@@ -0,0 +1,103 @@
+puts "How many employees will be processed?"
+n = gets.chomp.to_i
+
+while n > 0
+  puts "Thank you for taking our new employee survey!"
+  
+  # Questions to collect data from the employee.
+
+  puts "What is your name?"
+  name = gets.chomp
+  
+  puts "How old are you, or how old will you be turning this year?"
+  age = gets.chomp.to_i
+  
+  puts "What year were you born?"
+  birth_year = gets.chomp.to_i
+  by_age = 2017 - birth_year
+  
+  puts "Our company cafeteria serves garlic bread. Should we order some for you? (y/n)"
+  gbread = gets.chomp
+  
+  if gbread == "y"
+    gbread = true
+  elsif gbread == "n"
+    gbread = false
+  end
+  
+  puts "Would you like to enroll in the company's health insurance?"
+  health_ins = gets.chomp
+  
+  if health_ins == "y"
+    health_ins = true
+  elsif health_ins == "n"
+    health_ins = false
+  end
+
+  puts "Please input any allergies you have. Type done when complete."
+  sun_allergy = false
+
+  loop do
+    allergy = gets.chomp
+    if allergy == "sunshine"
+      sun_allergy = true
+      break
+    elsif allergy == "done"
+      break
+    end
+  end
+  
+  #If statements to analyze responses to determine likelihood of vampireness.
+  
+  result = "Results Inconclusive."
+  
+  if age == by_age && (gbread || health_ins)
```

Comment:
I'd remove the logic for checking `age == by_age` - this is something you can evaluate before and set to a boolean like `gbread` and `health_ins`